2.5.9

Tiakin opened the pull request:
- ServerVersion named properly
- Reflection on teleportSeat 
- Better mouvement selector by choosing it directly from the enum (and put the Mouvement on the enum)
- Select ServerVersion on load and keep it for the Getter (more efficient)
- **Huge** refactoring of PacketHandler
- Support for 1.14 to 1.14.4, 1.21.11, 26.1 to 26.2
- update dependencies
- bump to 2.5.9

Tiakin committed:
Rename variables and methods for clarity

Tiakin committed:
Using movement handlers in ServerVersion enum

Tiakin committed:
Update movement handler method names

Tiakin committed:
Merge remote-tracking branch 'upstream/master'

Tiakin committed:
Merge remote-tracking branch 'upstream/master'

Tiakin committed:
Merge branch 'master' into Refactor

Tiakin committed:
Refac: improved ServerVersion usage on VersionModule

Tiakin committed:
Refac: simplify teleportSeat method using reflection for server version handling

Tiakin committed:
Fix: update packet handling to re-use movement_1_12_R1 on reload

Tiakin committed:
Refactor: packet handling methods into a generic method

Tiakin committed:
Merge branch 'MTVehicles:master' into master

Tiakin committed:
feat: update dependencies

Tiakin committed:
feat: add support for 1.21.11

Tiakin committed:
refactor: deleted version check methods

Tiakin committed:
Bump to 2.5.9 and bump dependencies to latest versions

Tiakin committed:
feat: add support for 26.1

NicDevTV reviewed (approved):
Einfach mal getestet

NicDevTV reviewed (approved):
All changes are good executed and will be a good PR for the Repository.

Tiakin committed:
feat: add support for 1.14.X

Tiakin committed:
feat: add support for 26.1.2

Tiakin committed:
refactor: update dependencies

Updated the paper-api dependency to build with Java 8 and cleaned up Spigot dependencies as they have become unnecessary

Tiakin committed:
Merge branch 'master' into master

Tiakin committed:
Update dependency-reduced-pom.xml

Tiakin committed:
fix: prevent entering vehicles in spectator mode

Tiakin committed:
chore: update dependencies

Tiakin committed:
refactor: refine supported versions list

Tiakin committed:
refactor: use properties for pom.xmll

Tiakin committed:
fix: change JDK version from temurin to openjdk

Tiakin committed:
chore: update Skript dependency version to 2.15.2

Tiakin committed:
Update compatibility information in README

Tsoccerguy3 commented:
<img width="1360" height="768" alt="2026-06-19_04 42 12" src="https://github.com/user-attachments/assets/3a87421c-d134-404f-8c93-afc12c898751" />

Tiakin committed:
chore: update Skript dependency version to 2.15.3

Tiakin committed:
feat: add support for 26.2

I renamed "movement_26.1" to "movement_modern" because I don't think that will change over time

GomorrhaDev committed:
Remove unmaintaned repo

GomorrhaDev committed:
Add Pagination to the VehicleMenu

Tiakin committed:
refactor: added missing and clean up imports in InventoryClickListener

Tiakin committed:
Update dependency-reduced-pom.xml

Tiakin committed:
chore: update Skript dependency version to 2.15.4

Tiakin committed:
chore: update Skript dependency version to 2.16.0
